Tests/scripts/gitlab_slack_notifier.py

```diff
@@ -12,37 +12,35 @@
 
 DEMISTO_GREY_ICON = 'https://3xqz5p387rui1hjtdv1up7lw-wpengine.netdna-ssl.com/wp-content/' \
                     'uploads/2018/07/Demisto-Icon-Dark.png'
-ARTIFACTS_FOLDER = os.getenv('ARTIFACTS_FOLDER', './artifacts')
-ARTIFACTS_FOLDER_XSOAR = os.getenv('ARTIFACTS_FOLDER_XSOAR', './artifacts')
-ENV_RESULTS_PATH = os.getenv('ENV_RESULTS_PATH', os.path.join(ARTIFACTS_FOLDER_XSOAR, 'env_results.json'))
-PACK_RESULTS_PATH = os.path.join(ARTIFACTS_FOLDER_XSOAR, BucketUploadFlow.PACKS_RESULTS_FILE)
+ROOT_ARTIFACTS_FOLDER = os.getenv('ARTIFACTS_FOLDER', './artifacts')
+ARTIFACTS_FOLDER_XSOAR = os.getenv('ARTIFACTS_FOLDER_XSOAR', './artifacts/xsoar')
+ARTIFACTS_FOLDER_MPV2 = os.getenv('ARTIFACTS_FOLDER_MPV2', './artifacts/marketplacev2')
 CONTENT_CHANNEL = 'dmst-content-team'
 GITLAB_PROJECT_ID = os.getenv('CI_PROJECT_ID') or 2596  # the default is the id of the content repo in code.pan.run
 GITLAB_SERVER_URL = os.getenv('CI_SERVER_URL', 'https://code.pan.run')  # disable-secrets-detection
 CONTENT_NIGHTLY = 'Content Nightly'
 BUCKET_UPLOAD = 'Upload Packs to Marketplace Storage'
+BUCKET_V2_UPLOAD = 'Upload Packs to Marketplace v2 Storage'
 SDK_NIGHTLY = 'Demisto SDK Nightly'
 PRIVATE_NIGHTLY = 'Private Nightly'
-WORKFLOW_TYPES = {CONTENT_NIGHTLY, SDK_NIGHTLY, BUCKET_UPLOAD, PRIVATE_NIGHTLY}
+WORKFLOW_TYPES = {CONTENT_NIGHTLY, SDK_NIGHTLY, BUCKET_UPLOAD, PRIVATE_NIGHTLY, BUCKET_V2_UPLOAD}
 
 
 def options_handler():
     parser = argparse.ArgumentParser(description='Parser for slack_notifier args')
     parser.add_argument('-u', '--url', help='The gitlab server url', default=GITLAB_SERVER_URL)
+    parser.add_argument('-gp', '--gitlab_project_id', help='The gitlab project id', default=GITLAB_PROJECT_ID)
+    parser.add_argument(
+        '-tw', '--triggering-workflow', help='The type of ci pipeline workflow the notifier is reporting on',
+        choices=WORKFLOW_TYPES)
     parser.add_argument('-p', '--pipeline_id', help='The pipeline id to check the status of', required=True)
-    parser.add_argument('-s', '--slack_token', help='The token for slack', required=True)
     parser.add_argument('-c', '--ci_token', help='The token for circleci/gitlab', required=True)
-    parser.add_argument(
-        '-f', '--env_results_path', help='The env results file containing the dns address', default=ENV_RESULTS_PATH
-    )
-    parser.add_argument('-ca', '--ci_artifacts', help="The path to the ci artifacts directory")
+
+    parser.add_argument('-s', '--slack_token', help='The token for slack', required=True)
     parser.add_argument(
         '-ch', '--slack_channel', help='The slack channel in which to send the notification', default=CONTENT_CHANNEL
     )
-    parser.add_argument('-gp', '--gitlab_project_id', help='The gitlab project id', default=GITLAB_PROJECT_ID)
-    parser.add_argument(
-        '-tw', '--triggering-workflow', help='The type of ci pipeline workflow the notifier is reporting on',
-        choices=WORKFLOW_TYPES)
+
```

Comment:
Why the reordering? You are effectively only removing the `env_results_path` argument.

Reply:
I changed the arguments, so I separated them into groups.
I reverted most of them, and this is the result - I will revert to the original state.